### ISSUE TITLE
Tk103B not cut oil

### ISSUE THREAD

**Issue by mertturkman** [vitalidze/traccar-web#772]
I have Tk103b and it use GT06 protocol I send command for cutting oil over internet but it don't cut oil. if I send sms to device, cut oil . What must i do ?

**Comment by grafviktor** [comment]
Please check your tracker-server.log, if you see any errors when sending the command.

**Comment by mertturkman** [comment]
I don't see error but see this log
'2016-07-22 11:10:08  INFO: [43B514CB] command type: engineStop, id : 1 sent'

**Comment by grafviktor** [comment]
Have you tried other commands ?

**Comment by mertturkman** [comment]
Yes I send Resume and stop engine command but When I try sending another commands ,I see response. 
`"{\"reason\":\"java.lang.RuntimeException: Command positionSingle is not supported in protocol gt06\",\"success\":false}"`

**Comment by grafviktor** [comment]
How do you exactly send this `cut oil` command, via `Command -> type: custom` ? Or this is `Engine Stop` command, that actually isn't working ?

**Comment by mertturkman** [comment]
I send `two command` via `Command -> type: custom` for cut oil and I send `engine stop` command but Engine isn't stopped.

**Comment by grafviktor** [comment]
mertturkman, can you try to stop the engine by sending the command via SMS ? Is that working ?

**Comment by mertturkman** [comment]
yes I tried to stop engine via sms and worked. Engine is stopped.

**Comment by grafviktor** [comment]
**mertturkman**, please open a new issue in the [backend project](https://github.com/tananaev/traccar). Judging by the line from your log file, everything is working fine in the web-ui. I believe it can be a problem in the protocol or in your device itself.

**Comment by mertturkman** [comment]
I think Problem is in the protocol. I examanied , system send command `Relay,1#` to my device . According to me this command is not compatible for TK103B. I cant find working commands for TK103B.
Do you know stop and resume engine commands for TK103B? or Where are these commands ?
